src/api/users/controllers/update-user.js

```diff
@@ -46,7 +47,9 @@ const updateUserController = {
       }
     }
 
+    const lock = await requireLock(request.locker, 'users')
```

Comment:
does this need to be in an `if await requireLock(...)` statement so it only runs when  the lock is aquired, or does it just block until the lock is available?

Reply:
There are two helper methods. `acquireLock` and `requireLock`. `acquire` you need to check if lock is thruthy, whilst `require` will blow up if not able to get a lock.

Reply:
Both will wait until a lock is acquired or not at all